src/storage/mvcc/txn.rs

```diff
@@ -386,18 +387,7 @@ impl<S: Snapshot> MvccTxn<S> {
                 last_lock_ttl = lock.ttl;
             }
         } else if is_pessimistic_lock {
-            // Pessimistic lock does not exist, the transaction should be aborted.
-            warn!(
-                "prewrite failed (pessimistic lock not found)";
-                "start_ts" => self.start_ts,
-                "key" => %key
-            );
-
-            return Err(ErrorInner::PessimisticLockNotFound {
-                start_ts: self.start_ts,
-                key: key.into_raw()?,
-            }
-            .into());
+            self.is_amendable_perssimistic_lock(pipelined_pessimistic_lock, &key)?;
```

Comment:
I think `amend_pessimistic_lock` is a better name since `is_xxx` always returns boolean not a `Result`.

Reply:
BTW, there's a typo pe'r'ssimistic.

Reply:
My bad ...